docs/en/05_Contributing/13_CSS_Coding_Conventions.md

```diff
@@ -22,14 +22,18 @@ We use [Bootstrap 4](http://v4-alpha.getbootstrap.com/) styles where possible.
 
 We follow the [AirBnB CSS Conventions](https://github.com/airbnb/css)
 and the [BEM](http://getbem.com/) methodology (block-element-modifier).
-File naming and style include ordering is inspired by
-[ITCSS](https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/).
+
+Because we use [Bootstrap 4](http://v4-alpha.getbootstrap.com/) which 
```

Comment:
Could we update the bootstrap link? https://getbootstrap.com/
v4 is stable and the version we're using is not that far from the stable release